examples/vacuum_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build !experimental
```

Comment:
Is there any specific reason *not* to run this with experimental features on?

Reply:
yes, getting this result:

```
--- FAIL: ExampleVacuumSparseArray (0.08s)
got:
Estimated query size in bytes for attribute 'a': 12
Estimated query size in bytes for dimension 'd': 12
Cell (1) has data 1
Cell (2) has data 2
Cell (3) has data 3
Num of fragments after 2 writes before consolidate: 3
Num of fragments after consolidate: 4
Num of fragments after vacuum: 2
Estimated query size in bytes for attribute 'a': 12
Estimated query size in bytes for dimension 'd': 12
Cell (1) has data 1
Cell (2) has data 2
Cell (3) has data 3
want:
Estimated query size in bytes for attribute 'a': 12
Estimated query size in bytes for dimension 'd': 12
Cell (1) has data 1
Cell (2) has data 2
Cell (3) has data 3
Num of fragments after 2 writes before consolidate: 2
Num of fragments after consolidate: 3
Num of fragments after vacuum: 1
Estimated query size in bytes for attribute 'a': 12
Estimated query size in bytes for dimension 'd': 12
Cell (1) has data 1
Cell (2) has data 2
Cell (3) has data 3
```

Reply:
Since it's not obvious why this code should not be run with experimental flags (i.e., it currently fails), I would add a comment (after the build tag) that explains why it is disabled in that situation. Eventually, we should be able to remove the not-experimental constraint.